optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/rest/pagination/PaginationScrollableRequestDto.java

```diff
@@ -12,35 +12,26 @@
 import jakarta.validation.constraints.Max;
 import jakarta.validation.constraints.Min;
 import jakarta.ws.rs.DefaultValue;
-import jakarta.ws.rs.QueryParam;
 
 public class PaginationScrollableRequestDto {
-
-  public static final String QUERY_LIMIT_PARAM = "limit";
-  public static final String QUERY_SCROLL_ID_PARAM = "searchRequestId";
-  public static final String QUERY_SCROLL_TIMEOUT_PARAM = "paginationTimeout";
-
-  @QueryParam(QUERY_LIMIT_PARAM)
   @Min(0)
   @DefaultValue("1000")
   @Max(MAX_RESPONSE_SIZE_LIMIT)
   protected Integer limit;
 
-  @QueryParam(QUERY_SCROLL_ID_PARAM)
-  protected String scrollId;
+  protected String searchRequestId;
```

Comment:
❓ The BE changes for this DTO change look correct, but does the FE need changing as well?

Reply:
Solved my own question here. It was renamed here due to the removal of the @QueryParam annotation, and the ultimate DTO that is used will have the same name for fields